packages/model-viewer/src/features/controls.ts

```diff
@@ -340,6 +341,9 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
     @property({type: Boolean, attribute: 'disable-zoom'})
     disableZoom: boolean = false;
 
+    @property({type: Number, attribute: 'interpolation-decay-milliseconds'})
```

Comment:
I don't think we need milliseconds in the name here, since that's the JS default time unit anyway (we don't bother for `auto-rotate-delay` for instance). How about `interpolation-decay-time`?